src/pages/Issue/Summary.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Grid, Section } from '@digicatapult/ui-component-library'
+import { Button } from '../../components/shared'
+
+const Row = styled('div')`
+  display: flex;
+  flex-direction: row;
+  justify-content: ${({ justify }) => justify || 'center'};
+`
+
+const WarnBox = styled('div')`
+  display: flex;
+  height: 28px;
+  flex-direction: row;
+  justify-content: center;
+  flex-shrink: 0;
+  color: #000;
+  font-family: Roboto Mono;
+  font-size: 6px;
+  font-style: italic;
+  font-weight: 400;
+  line-height: normal;
+`
+
+const Th = styled('th')`
+  font-style: italic;
+  font-size: 8px;
+`
+
+const Td = styled('td')`
+  font-size: 8px;
+  align-text: center;
+`
+
+export default function Summary({ inputs, setStage, handleSubmit }) {
```

Comment:
I know this is a basic layout for mobile but this summary is really small at my laptop resolution?
<img width="2006" alt="image" src="https://github.com/digicatapult/veritable-portal/assets/40722025/525d7978-9441-4c3e-8c7f-b178c8f023df">

Reply:
updated

Reply:
yep looks better thanks